app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -120,21 +113,17 @@ def instance_storage(persister_hardware, instance)
 
   def instance_operating_system(persister_instance, instance)
     image_id = instance&.dig(:image, :id)
-    os = img_to_os[image_id] || pub_img_os(image_id)
-    os_name = normalize_os(os)
-    persister.operating_systems.build(
-      :vm_or_template => persister_instance,
-      :product_name   => os_name,
-      :version        => os
-    )
+    image    = collector.images_by_id[image_id] || collector.image(image_id) if image_id
+
+    vm_or_template_operating_system(persister_instance, image) if image
   end
 
-  def image_operating_system(persister_image, image)
+  def vm_or_template_operating_system(vm_or_template, image)
```

Comment:
Is the rename of the method to `vm_or_template_operating_system` a best practice or standard used in other providers?

Reply:
Yeah since this is now used by both images and instances I thought it better to name the method for the manager_ref (vm_or_template) [down here :point_down:](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/115/files#diff-be65168162a51f26d1c4e78ad610a312bea5fbefd1cff9fb19152ef1e89b9992R125)